src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Unix.cs

```diff
@@ -110,14 +158,26 @@ public ClientCertificateOption ClientCertificateOptions
                 }
                 else
                 {
-                    _managedHandler.ServerCertificateCustomValidationCallback = value;
+                    ThrowForModifiedManagedSslOptionsIfStarted();
+                    if (value == null)
+                    {
+                        _managedHandler.SslOptions.RemoteCertificateValidationCallback = null;
+                    }
+                    else
+                    {
+                        ServerCertificateValidationCallbackManagedMapper mapper =
+                            _managedHandler.SslOptions.RemoteCertificateValidationCallback?.Target as ServerCertificateValidationCallbackManagedMapper ??
+                            new ServerCertificateValidationCallbackManagedMapper();
+                        mapper.ServerCertificateCustomValidationCallback = value;
+                        _managedHandler.SslOptions.RemoteCertificateValidationCallback = mapper.RemoteCertificateValidationCallback;
+                    }
                 }
             }
         }
 
         public bool CheckCertificateRevocationList
         {
-            get => _curlHandler != null ? _curlHandler.CheckCertificateRevocationList : _managedHandler.CheckCertificateRevocationList;
+            get => _curlHandler != null ? _curlHandler.CheckCertificateRevocationList : _managedHandler.SslOptions.CertificateRevocationCheckMode == X509RevocationMode.Online;
```

Comment:
Is `Offline` capable of happening anywhere?  It seems weird that `Offline` would mean "no".

Reply:
Not via HttpClientHandler; the only place it's set is from the setter, which uses either Online or NoCheck.